init-odkx-sync-endpoint.py

```diff
@@ -57,73 +61,73 @@ def run_interactive_config():
 
     return (enforce_https, env)
 
-def run_cache_setup(envParam) -> bool:
+def run_cache_setup(envParam: Dict[str, str]) -> bool:
 
-    env:Dict[str, str] = envParam
```

Comment:
Unrelated change in the commit. Okay for now, but in general it's better to try and keep each commit isolated to one specific set of changes.